pack/tdc/tdcp.json

```
@@ -1423,6 +1423,27 @@
         "type_code": "asset",
         "xp": 5
     },
+    {
+        "code": "11121",
+        "deck_limit": 1,
+        "cost": 1,
+        "faction_code": "neutral",
+        "illustrator": "Aleksander Karcz",
+        "name": "Sacred Oath",
+        "pack_code": "tdcp",
+        "position": 121,
+        "quantity": 1,
+        "restrictions": "trait:cultist, trait:cursed, trait:'silver twilight'",
```

```suggestion
        "restrictions": "trait:cultist, trait:cursed, trait:silver twilight",
```

I think we should expect this to be split via `, `, so just lowercase should be fine. cc @zzorba / @drawntotheflame ?